app/assets/stylesheets/scaffolds.scss

```diff
@@ -1,10 +1,10 @@
 body {
   background-color: #fff;
   color: #333;
-  margin: 33px;
-  font-family: verdana, arial, helvetica, sans-serif;
+  margin: 10px;
+  font-family: Comic Sans MS, verdana, arial, helvetica, sans-serif;
```

Comment:
a different font?

Reply:
Why does anyone hates Comic Sans so much??
Changed it..